lnrpc/marshall_utils.go

```diff
@@ -179,3 +180,57 @@ func MarshalOutPoint(op *wire.OutPoint) *OutPoint {
 		OutputIndex: op.Index,
 	}
 }
+
+// Parse the coin selection strategy string from the command line
+// to an equivalent lnrpc.CoinSelectionStrategy enum type.
+func ParseCoinSelectionStrategyString(strategy string)(
```

Comment:
nit: please check our formatting guidelines: https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md

Reply:
Also, the "parse" is usually used to parse strings. So the function name doesn't need to have the word "string" in it.

Reply:
Thanks, I missed this formatting guideline comment:
```
Function comments must always begin with the name of the function per [Effective Go](http://golang.org/doc/effective_go.html)
```